docs/concepts/copilot/copilot.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Data Copilot
+id: copilot
+description: The AI assistant for data Pipelines and models
+sidebar_position: 8
+tags:
+  - concepts
+  - copilot
+  - generativeai
+---
+
+Prophecy’s end-to-end platform makes data Pipeline development faster and accessible to more users. On top of that, you have an AI assistant to boost that productivity even further.
+
+## Building Models and Pipelines with Prophecy’s Data Copilot
+
+<div class="wistia_responsive_padding" style={{padding:'62.5% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/t7m3boc575?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
```

Comment:
Not urgent, but we should create a new video to show overall capabilities of copilot now that we have lot of features. We can use Maciej's DAIS demo script and show a full lifecycle of development using copilot.

Reply:
Got it. Added as a future task.